src/org/usfirst/frc1735/Steamworks2017/Robot.java

```diff
@@ -25,6 +25,8 @@
  * creating this project, you must also update the manifest file in the resource
  * directory.
  */
+
+//FRC1735-test comment
```

Comment:
In reviewing this comment, I'd like to see it have a space between the comment characters and the actual comment text (good coding style guidelines)